docs/DOCKER.rst

```diff
@@ -280,3 +295,51 @@ For instance, you can modify these lines in the dockerfile:
 
     USER scality
     ENTRYPOINT ["/usr/src/app/docker-entrypoint.sh"]
+
+Continuous integration with Docker hosted CloudServer
+-----------------------------------------------------
+
+When you start the Docker Scality Zenko CloudServer image, you can adjust the
+configuration of the Scality Zenko CloudServer instance by passing one or more
+environment variables on the docker run command line.
+
+Sample ways to run it for CI are:
+
+- With custom locations, including one hosted on AWS, and custom credentials mounted:
+
+.. code:: shell
+
+    docker run --name CloudServer -p 8000:8000
+    -v $(pwd)/locationConfig.json:/usr/src/app/locationConfig.json
+    -v $(pwd)/authdata.json:/usr/src/app/conf/authdata.json
+    -v ~/.aws/credentials:/root/.aws/credentials
+    -e S3DATA=multiple -e S3BACKEND=mem scality/s3server
```

Comment:
shouldn't you be mounting the data and metadata directories as well?

Reply:
Well, this section is for CI and refers to a usecase with S3BACKEND=mem, so I wouldn't mount data / md volumes here... (however, S3DATA is set to multiple to enable testing against multiple backends). Does it make sense ?

Reply:
For CI I guess it does not make sense but for the example below (custom locations) it seems we should.

Reply:
I've rephrased to clarify as they're both CI examples, but I've mounted volumes in one of them with a file usecase.